eox_tagging/api/v1/viewset.py

```diff
@@ -29,11 +29,11 @@ class TagViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         """Restricts the returned tags."""
         owner_type = self.request.query_params.get("owner_type")
-        include_invalid = self.request.query_params.get("include_invalid")
+        include_inactive = self.request.query_params.get("include_inactive")
```

Comment:
Where are you documenting this changes ??? why there are no tests for this,  why you can change this so easy ?

Reply:
The tests are in this PR https://github.com/eduNEXT/eox-tagging/pull/18
I found this error while testing using postman. I'll will close this PR and change this in #18